package.json

```diff
@@ -13,18 +15,20 @@
     "yext"
   ],
   "exports": {
-    "import": "./lib/esm/index.js",
-    "require": "./lib/commonjs/index.js"
+    "import": "./dist/index.js",
+    "import.types": "./dist/index.d.ts",
+    "require": "./dist/index.cjs",
+    "require.types": "./dist/index.d.cts"
   },
   "files": [
     "lib",
+    "dist",
     "src",
     "THIRD-PARTY-NOTICES"
   ],
   "scripts": {
-    "build": "rm -rf lib/** && npm run build:esm && npm run build:cjs && npm run generate-notices",
-    "build:esm": "tsc -p tsconfig.esm.json",
-    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build": "rm -rf lib/** && rm -rf dist/** && npm run build:js && npm run generate-notices",
+    "build:js": "tsc -p tsconfig.esm.json && tsc -p tsconfig.cjs.json && tsup src/index.ts --dts --sourcemap --format esm,cjs",
```

Comment:
I still don't understand the tsc thing. Your argument was that people can import the cjs/esm files specifically at lib/cjs or lib/esm. Doesn't that mean you're effectively building this 4 times now? Twice with tsc and twice with tsup.

Reply:
yes, that means the it's being built 4 times. If we want to remove two of those, I can remove tsc and copy the build output from dist to lib, or I could add additional package.json files in lib in order to redirect those imports to dist

Reply:
I removed tsc and copied the files into lib